content/200-concepts/100-components/03-prisma-migrate/200-shadow-database.mdx

```diff
@@ -26,6 +19,8 @@ The shadow database is **created and deleted automatically**\* each time you run
 
 > **Important**: The shadow database is **not** required in production, and is not used by production-focused commands such as `prisma migrate resolve` and `prisma migrate deploy`.
 
+> **Note**: A shadow database is never used for MongoDB as these commands are not used there.
```

Comment:
Adds a highlight to tell people that the shadow database is never used here, just for ease of mind and to avoid potential confusion.